packages/acs-ui-common/package.json

```diff
@@ -11,6 +11,12 @@
     "build:cjs": "rollup -c --silent --failAfterWarnings",
     "build:watch": "rushx build",
     "clean": "rimraf dist",
+    "copy-original": "cpx ./src/** ./preprocessed",
+    "preprocess": "rushx copy-original && babel ./src --out-dir ../preprocessed --extensions \".ts,.tsx\" --keep-file-extension --config-file ./.babelrc.js --relative && rimraf ../preprocessed",
```

Comment:
Do we ever want to run `rushx preprocess` instead of `rushx preprocess:stable`? I'd drop this one as well.